### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by tarunsankhla**

### CODE REVIEW

File: src/JS/JS.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            <div class="quiz-question">
+                The 3 basic object attributes in Javascript are?
+            </div>
+            <div class="quiz-answer">
+                <button class="btn btn-warning">Class, prototype, objects' parameters.</button>
+                <button class="btn btn-sucess">Class, prototype, object's extensible flag.</button>
+                <button class="btn">Class, parameters, object's extensible flag.</button>
+                <button class="btn btn-sucess">Classes, Native object, and Interfaces and Object's extensible flag.</button>
+              </div>
+        </div>
```
Comment (theVivek-kumar) [inline src/JS/JS.html:76]: it looks good to me even space and naming

File: style.css [hunk 2]
```diff
@@ -99,4 +103,23 @@ body::-webkit-scrollbar:hover{
...
+}
+
+.card{
+    background-color: #1E293B;
+    flex-grow: 1;
+    /* width: 40%; */
+    margin: 1rem 3rem;
+    height: 50vh;
+    text-decoration: none;
+    color: white;
```
Comment (theVivek-kumar) [inline style.css:117]: this is good you use rem  and I am also doing the same practice  but use color variable

**Review by theVivek-kumar — APPROVED** [review]
everything looks good to me

**Review by Debarshi95 — COMMENTED** [review]
Looks good to me. A suggestion though, try to use semantic html tags instead of creating div soup.

File: style.css (hunk 2 above)
Comment (tarunsankhla) [reply style.css:117]: sure

**Comment by tarunsankhla** [comment]
added the review changes

**Pull request merged**